vnext/Microsoft.ReactNative/Modules/Animated/PropsAnimatedNode.cpp

```diff
@@ -165,22 +166,25 @@ void PropsAnimatedNode::DisposeCompletedAnimation(int64_t valueTag) {
       // Fixing this will prevent memory bloat as the current solutions never
       // stops the expression animations that are built as a part of this
       // animation solution. Tracked by issue #3280.
-      // target.StopAnimation(m_expressionAnimations.at(valueTag));
+      target.StopAnimation(m_expressionAnimations.at(valueTag));
       m_suspendedExpressionAnimationTags.push_back(valueTag);
     }
   }
+  */
 }
 
 void PropsAnimatedNode::ResumeSuspendedAnimations(int64_t valueTag) {
+  /*
```

Comment:
Same question here. Should this be deleted?

Reply:
I was following how the previous change was made, where the functionality to suspend animations was just commented out.  It sounds like there is a general consensus (that I agree with) that the code should have been deleted, so I'll go ahead and do that.